### PR TITLE
Fix the bug that command line doesn't support regx file filter

### PR DESCRIPTION
**Pull request by JiamingMai**
Fix the bug that command line doesn't support regx file filter

### CODE REVIEW

**Review by elega — APPROVED** [review]

**Comment by JiamingMai** [comment]
alluxio-bot, merge this please.

**Comment by JiamingMai** [comment]
alluxio-bot, merge this please.

**Comment by JiamingMai** [comment]
alluxio-bot, force merge this please

**Comment by JiamingMai** [comment]
alluxio-bot, merge this please.

**Comment by JiamingMai** [comment]
alluxio-bot, merge this please.

**Comment by JiamingMai** [comment]
alluxio-bot, cherry-pick this to branch-3-1.1.x please.

File: cli/src/alluxio.org/cli/cmd/job/load.go [hunk 1]
```diff
@@ -12,46 +12,46 @@
 package job
 
 import (
```
Comment (Xenorith) [inline cli/src/alluxio.org/cli/cmd/job/load.go:14]: @JiamingMai @elega
please don't approve or merge PRs that reformat the entire file unless the purpose of the PR is to format. in addition to causing the back and forth formatting changes that will likely happen in the future, it also breaks git history tracking of who modified which line

**Pull request merged**